src/main.cc

```diff
@@ -178,6 +178,37 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_sigint_cb(ev_loop *loop, struct ev_signal *w, int revents)
+{
+	(void) loop;
+	(void) w;
+	(void) revents;
+
+	/**
+	 * If running in daemon mode, tarantool exits on SIGINT,
+	 * complaining about possibly sudden and unexpected death.
```

Comment:
I guess an operation team complains, not tarantool :)

Reply:
Fixed :)